virt/virt-4-10-release-notes.adoc

```diff
@@ -114,6 +114,8 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 
 * You can now link:https://access.redhat.com/articles/6409731[deploy {VirtProductName} on AWS bare metal nodes].
 
+* {VirtProductName} now has xref:../virt/logging_events_monitoring/virt-virtualization-alerts[critical alerts] that inform you when a problem occurs that requires immediate attention. Each alert has a corresponding description of the problem, a reason for why the alert is occurring, a troubleshooting process to diagnose the source of the problem, and steps for resolving the alert.
+
```

Comment:
@mgarrellRH 
* {VirtProductName} *has* xref:../virt/logging_events_monitoring/virt-virtualization-alerts[critical alerts] that inform you when a problem occurs that requires immediate attention. *Now* each alert has a corresponding description of the problem, a reason for why the alert is occurring, a troubleshooting process to diagnose the source of the problem, and steps for resolving the alert.